addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/internal/config/ConfigHelper.java

```diff
@@ -367,7 +367,7 @@ public Address addAddress(Object element, Item item, Transform transform, String
                 return address;
             }
         } catch (Exception e) {
-            logger.error(e.getMessage());
+            logger.error("{}", e.getMessage());
```

Comment:
16x I would log the exception instead, it is an error, I want to be able to debug the stacktrace